gp-settings.php

```diff
@@ -129,6 +129,11 @@
 	gp_upgrade_db();
 }
 
+/**
+ * Run the plugin activation code.
+ *
+ * @since    0.1
```

Comment:
There should be only one space between the `@since` tag and the version.

See also https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#since-section-changelogs.